onnxruntime/core/providers/dml/DmlExecutionProvider/inc/IWinmlExecutionProvider.h

```diff
@@ -73,19 +76,15 @@ namespace Windows::AI::MachineLearning::Adapter
         std::unique_ptr<AbstractOperatorDesc> desc;
     };
 
-    // This is the counterpart to the MLOperatorKernelDmlProperties ABI struct which owns its memory and uses containers.
+    // This is the counterpart to the MLOperatorGraphDesc ABI struct which owns its memory and uses containers.
     struct DmlGraphNodeCreateInfo
     {
-        bool initialized = false;
-
-        // Mapping between DML in/out indices and kernel in/out indices
-        std::vector<uint32_t> kernelInputIndices;
-        std::vector<uint32_t> kernelOutputIndices;
-
-        Microsoft::WRL::ComPtr<IDMLOperator> op;
-        std::unique_ptr<AbstractOperatorDesc> desc;
-
-        bool allowHalfPrecisionComputation = false;
```

Comment:
>bool allowHalfPrecisionComputation = false; [](http://example.com/codeflow?start=8&length=43)

Hmm, is this unused now? (orphaned stale code?)

Reply:
Yes. I did search for it the entire repo.